default.nix

```diff
@@ -305,6 +305,15 @@ rec {
         buildArgs = "-A overview --show-trace -v";
       })
     ];
+    shellHook = ''
+      ngipkgs-test() {
+        local pr="$1"
+        local proj="$2"
+        local test="$3"
```

Comment:
would be cool if it could tell you which tests are available if you forget that one (we can leave this as an in-code comment, not needed now)

Reply:
Yeah I agree. 

One quirk I noticed was that when using `nix-build -A` you can just do `<project>.tests` and it'll build them all. If you do `nix build .checks.....<project>/tests` it errors. You have to specifcy the test.